### ISSUE TITLE
The README isn't very descriptive

### ISSUE THREAD

**Issue by cosmojg** [joalon/vimway#2]
As an avid user of both Sway and Neovim, what little information I can find about Vimway excites me! Unfortunately, I'm still not very sure what Vimway *is* or *does* outside of being a Neovim-inspired Wayland compositor.

What is the Neovim expression engine? How does Vimway use it? How does Vimway look and feel? What differentiates Vimway from compositors like Sway and Way-Cooler? Got any pictures or videos?

In short, I'm pretty excited about this project, and I'm dying to know more! I think fleshing out the README would be a good place to start.
